healthtools/scrapers/brand_names.py

```diff
@@ -0,0 +1,136 @@
+from bs4 import BeautifulSoup
+from cStringIO import StringIO
+from datetime import datetime
+from healthtools.config import AWS
+import requests
+import boto3
+import re
+import json
+import hashlib
+
+
+class BrandNamesScraper(object):
```

Comment:
Let's change this to `MedBrandNamesScraper` to be clear that it's for Medicines.